sklearn/linear_model/_logistic.py

```diff
@@ -807,7 +813,7 @@ def _log_reg_scoring_path(
 class LogisticRegression(LinearClassifierMixin, SparseCoefMixin, BaseEstimator):
     """
     Logistic Regression (aka logit, MaxEnt) classifier.
-
+    
```

Comment:
Please remove these whitespaces. As you can see the linter is complaining about this. See also https://scikit-learn.org/dev/developers/contributing.html#how-to-contribute, especially step 10 about setting up pre-commit hooks for style check.